t/config.t

```diff
@@ -157,6 +162,28 @@ subtest 'Test configuration override from file' => sub {
         ],
         'referers parsed correctly'
     );
+    my $untracked_assets_patterns = $app->config->{'assets/storage_duration'};
+
+    ok(ref $untracked_assets_patterns eq ref {}, "failed to read config for assets/storage_duration");
```

Comment:
the test string "failed to read" reads a bit weirdly. Can you rephrase this so that it would not confuse a reader of the test output?

Reply:
What is this test supposed to do? Why does a hashref signal a failed read? Why `eq ref {}` instead of `eq 'HASH'`?